src/main/java/at/hannibal2/skyhanni/features/combat/damageindicator/MobFinder.kt

```diff
@@ -223,7 +223,7 @@ class MobFinder {
     }
 
     private fun tryAddDungeonF5(entity: EntityLivingBase): EntityResult? {
-        if (entity is EntityOtherPlayerMP && entity == DungeonLividFinder.lividEntity) {
+        if (entity is EntityOtherPlayerMP && entity == DungeonLividFinder.livid?.baseEntity) {
```

Comment:
why is here not DungeonLividFinder.lividEntity getting used?

Reply:
its not being used because DungeonLividFinder.lividEntity can return the livid's armor stand if the base entity doesnt exist anymore, which is an unnecessary check here

Reply:
i feel like DungeonLividFinder.lividEntity should indicate that via comment or changed variable name